react-app/src/components/molecules/Forms/EditMovieForm.tsx

```diff
@@ -0,0 +1,166 @@
+      title: movie.title,
+      releaseDate: movie.yearOfRelease,
+      movieUrl: movie.img,
+      genre: movie.genre.map((item) => ({ value: item, label: item })),
+      overview: movie.subtitle,
+      runtime: movie.duration !== null ?? movie.duration.toString(),
+    },
+    validationSchema: schema,
+    onSubmit: (value: IMovieValues) => {
+      const updatedMovie: IMovie = {
+        id: Number(value.movieId),
+        title: value.title,
+        subtitle: movie.subtitle,
+        description: value.overview,
+        rate: movie.rate,
+        duration: Number(value.runtime),
+        img: value.movieUrl,
+        yearOfRelease: value.releaseDate,
+        genre: value.genre.map((item) => (item.value)),
+      };
+      dispatch(actionUpdateMovie(updatedMovie));
+      dispatch(actionControlVisibility('edit', false));
+    },
+  });
+
+  return (
+    <FormElement onSubmit={formik.handleSubmit}>
+      <FormTitle>EDIT MOVIE</FormTitle>
+      <FieldWrapper>
+        <FieldLabel htmlFor="movieId">Movie id</FieldLabel>
+        <FieldText
+          name="movieId"
+          type="text"
+          placeholder="MO32820TH"
+          onBlur={formik.handleBlur}
+          onChange={formik.handleChange}
+          value={formik.values.movieId}
+          readOnly
+        />
+      </FieldWrapper>
+      <FieldWrapper>
+        <FieldLabel htmlFor="title">Title</FieldLabel>
+        <FieldText
+          name="title"
+          type="text"
+          placeholder="Moana"
+          onBlur={formik.handleBlur}
+          onChange={formik.handleChange}
+          value={formik.values.title}
+        />
+        { formik.errors.title && formik.touched.title ? (
+          <FieldError>{formik.errors.title}</FieldError>
+        ) : null }
+      </FieldWrapper>
+      <FieldWrapper>
+        <FieldLabel htmlFor="releaseDate">Release date</FieldLabel>
+        <FieldText
+          name="releaseDate"
+          type="text"
+          placeholder="2018-03-31"
+          onBlur={formik.handleBlur}
+          onChange={formik.handleChange}
+          value={formik.values.releaseDate}
+        />
+        { formik.errors.releaseDate && formik.touched.releaseDate ? (
+          <FieldError>{formik.errors.releaseDate}</FieldError>
+        ) : null }
+      </FieldWrapper>
+      <FieldWrapper>
+        <FieldLabel htmlFor="movieUrl">Movie url</FieldLabel>
+        <FieldText
+          name="movieUrl"
+          type="text"
+          placeholder="www.moana.com"
+          onBlur={formik.handleBlur}
+          onChange={formik.handleChange}
+          value={formik.values.movieUrl}
+        />
+        { formik.errors.movieUrl && formik.touched.movieUrl ? (
+          <FieldError>{formik.errors.movieUrl}</FieldError>
+        ) : null }
+      </FieldWrapper>
+      <FieldWrapper>
+        <FieldLabel htmlFor="genre">Genre</FieldLabel>
+        <StyledSelect
+          id="genre"
+          name="genre"
+          isMulti
+          options={[
```

Comment:
I would rather suggest having movie genres as typescript enums and avoid defining such a massive object within components body. You could've done something, like:
```ts
export enum Genre {
   DOCUMENTARY = 'Documentary',
   COMEDY = 'Comedy',
   HORROR = 'Horror',
   CRIME = 'Crime',
   ACTION = 'Action'
}

const options = Object
   .values(Genre)
   .map(label=> ({
      label,
      value: label.toLowerCase()
   }))

<StyledSelect
   ...
   options={options}
```
This could've made more consistent and re-usable genres.